.changeset/wicked-planes-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+"@tokens-studio/graph-engine-nodes-design-tokens": patch
+---
+
+Fixes the incorrectly typed Tokenset Schema
```

Comment:
shouldn't this also include some info of the pipeline change?

Reply:
No, nothing within the packages changes from the pipeline change, this is only for customer facing info